Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && apt-get install -y php-cli php-mbstring git unzip && apt-get clean
 
+# github-releases/composer/composer
```

Comment:
Maybe using `renovate:` prefix to be more save?

Reply:
Yeah, if a line starts with `# renovate:` and the line after it starts with `ENV ` then that seems very safe

Reply:
you merged the composer pr. so now i can't test it 😅